src/caustics/parametrized.py

```diff
@@ -370,10 +369,11 @@ def params(self) -> NestedNamespaceDict:
         dynamic = NestedNamespaceDict()
 
         def _get_params(module):
-            if module.module_params.static:
-                static[module.name] = module.module_params.static
-            if module.module_params.dynamic:
-                dynamic[module.name] = module.module_params.dynamic
+            MP = module.module_params
```

Comment:
I don't agree with this artificial change. Capital letters should not be used as name variable except for global variables stated with the import statements. Please revert this code to its original form.

Reply:
Good catch, changed to lower case. The other modification just helps reduce the number of calls to recursive functions that re-build the same dictionary multiple times. At some point we may want to go through all of `parametrized` and see where we can optimize the calls